components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -140,20 +260,61 @@ const getTransactionsBreakdownChartData = host => {
   return areas;
 };
 
-const TransactionsOverviewSection = ({ host, isLoading }) => {
+const TransactionsOverviewSection = ({ host, isLoading, dateInterval }) => {
+  const intl = useIntl();
+  const categoryType = getCategoryType(dateInterval?.from, dateInterval?.to);
+
+  const contributionStats = host?.contributionStats;
+  const expenseStats = host?.expenseStats;
+
+  const { contributionAmountOverTime } = contributionStats || 0;
+  const { expenseAmountOverTime } = expenseStats || 0;
+
+  const series = [
+    {
+      name: 'Contributions',
+      data: contributionAmountOverTime
+        ? constructDataPointObjects(categoryType, contributionAmountOverTime.nodes)
+        : null,
+    },
+    {
+      name: 'Expenses',
+      data: expenseAmountOverTime ? constructDataPointObjects(categoryType, expenseAmountOverTime.nodes) : null,
+    },
+  ];
+
   const areaChartData = React.useMemo(() => getTransactionsAreaChartData(host), [host]);
   const transactionBreakdownChart = React.useMemo(() => getTransactionsBreakdownChartData(host), [host]);
   return (
-    <Box mt={18} mb={12}>
-      {isLoading ? (
-        <LoadingPlaceholder height="98px" borderRadius="8px" />
-      ) : (
-        <div>
-          <ProportionalAreaChart areas={areaChartData} borderRadius="6px 6px 0 0" />
-          <ProportionalAreaChart areas={transactionBreakdownChart} borderRadius="0 0 6px 6px" />
-        </div>
-      )}
-    </Box>
+    <React.Fragment>
+      <Box mt={18} mb={12}>
+        {isLoading ? (
+          <LoadingPlaceholder height="98px" borderRadius="8px" />
+        ) : (
+          <div>
+            <ProportionalAreaChart areas={areaChartData} borderRadius="6px 6px 0 0" />
+            <ProportionalAreaChart areas={transactionBreakdownChart} borderRadius="0 0 6px 6px" />
+          </div>
+        )}
+      </Box>
+      <Box>
+        {isLoading ? (
+          <Loading />
```

Comment:
For consistency, can we use a `LoadingPlaceholder` with the same size as the chart it's replacing?

Reply:
Done. 👍🏽